src/index.html

```diff
@@ -0,0 +1,210 @@
+                </li>
+            </ul>
+            <div class="button-resume">
+                <button type="button" class="btn">
+                    Get my resume
+                </button>
+            </div>
+        </div>
+        <div class="right">
+            <div class="lang">
+
+            </div>
+            <ul class="language-list">
+                <li class="lang">
+                    <p>
+                        Languages
+                    </p>
+                    <img class="imglang" src="../assets/arrowdown.png" alt="arrow">
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/js.png" alt="js">
+                    <p>
+                        JavaScript
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/css.png" alt="html">
+                    <p>
+                        CSS
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/html.png" alt="css">
+                    <p>
+                        HTML
+                    </p>
+                </li>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h2 class="contact-title">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                    <p class="alert"></p>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
+        </div>
+    </section>
+
+    <div class="modal-example">
```

Comment:
- [ ] [OPTIONAL] Kindly fix the overflow property, check the image below for confirmation.

 - Your design with overflow. 😀
![mobile](https://user-images.githubusercontent.com/41163571/222924928-2604239b-ef75-4f73-8c06-8f00d237a3fc.gif)